src/test_l1_dsVideoPort.c

```diff
@@ -1921,40 +1735,47 @@ void test_l1_dsVideoPort_positive_dsGetResolution(void) {
 	UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
 
 	dsError_t status;
-	intptr_t handle[NUM_OF_PORTS]={HANDLE_ARRAY_INIT}; 
+	intptr_t handle[gDSvideoPort_NumberOfPorts]; 
 
-	dsVideoPortResolution_t resolutionArray1[NUM_OF_PORTS];
-	dsVideoPortResolution_t resolutionArray2[NUM_OF_PORTS];
+	dsVideoPortResolution_t resolutionArray1[gDSvideoPort_NumberOfPorts];
+	dsVideoPortResolution_t resolutionArray2[gDSvideoPort_NumberOfPorts];
 
 	// Step 01: Initialize video port system
 	status = dsVideoPortInit();
 	UT_ASSERT_EQUAL(status, dsERR_NONE);
 
 	// Step 02: Get the video port handle
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
-		status = dsGetVideoPort(kPorts[i].id.type, kPorts[i].id.index, &(handle[i]));
+	for (int i = 0; i < gDSvideoPort_NumberOfPorts; i++) {
+		status = dsGetVideoPort(gDSVideoPortConfiguration[i].typeid, gDSVideoPortConfiguration[i].index, &(handle[i]));
 		UT_ASSERT_EQUAL(status, dsERR_NONE);
 	}
 
 	// Step 03: Retrieve the resolution of the video port
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
+	for (int i = 0; i < gDSvideoPort_NumberOfPorts; i++) {
 		status = dsGetResolution(handle[i], &(resolutionArray1[i]));
 		UT_ASSERT_EQUAL(status, dsERR_NONE);
 	}
 
 	// Step 04: Repeat the retrieval of resolution
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
+	for (int i = 0; i < gDSvideoPort_NumberOfPorts; i++) {
 		status = dsGetResolution(handle[i] , &resolutionArray2[i]);
 		UT_ASSERT_EQUAL(status, dsERR_NONE);
 	}
 
 	// Step 05: Compare the array values
-	for (int i = 0; i < NUM_OF_PORTS; i++) {
+	for (int i = 0; i < gDSvideoPort_NumberOfPorts; i++) {
+		for (int j = 0; i < gDSVideoPortConfiguration[i].numSupportedResolutions; j++) {
```

Comment:
not sure what value we get without set 
check for default values

Reply:
done